tests/sentry/relay/test_config.py

```diff
@@ -120,20 +129,50 @@ def test_get_experimental_config(mock_sentry_sdk, _, default_project):
     assert mock_sentry_sdk.capture_exception.call_args == mock.call(SOME_EXCEPTION)
 
 
+@pytest.mark.django_db
+@mock.patch("sentry.relay.config.capture_exception")
+def test_get_experimental_config_transaction_metrics_exception(
+    mock_capture_exception, default_project
+):
+    keys = ProjectKey.objects.filter(project=default_project)
+    default_project.update_option("sentry:breakdowns", {"invalid_breakdowns": "test"})
+    # wrong type
+    default_project.update_option("sentry:transaction_metrics_custom_tags", 42)
+
+    with Feature({"organizations:transaction-metrics-extraction": True}):
+        cfg = get_project_config(default_project, full_config=True, project_keys=keys)
+
+    # we check that due to exception we don't add `d:transactions/breakdowns.span_ops.ops.{op_name}@millisecond`
+    assert (
+        "breakdowns.span_ops.ops"
+        not in cfg.to_dict()["config"]["transactionMetrics"]["extractMetrics"]
+    )
+    assert cfg.to_dict()["config"]["transactionMetrics"]["extractCustomTags"] == []
+    assert mock_capture_exception.call_count == 2
+
+
 @pytest.mark.django_db
 @pytest.mark.parametrize("has_custom_filters", [False, True])
-def test_project_config_uses_filter_features(default_project, has_custom_filters):
+@pytest.mark.parametrize("has_blacklisted_ips", [False, True])
+def test_project_config_uses_filter_features(
+    default_project, has_custom_filters, has_blacklisted_ips
+):
     error_messages = ["some_error"]
     releases = ["1.2.3", "4.5.6"]
+    blacklisted_ips = ["112.69.248.54"]
```

Comment:
`blocked_ips` or `denied_ips` fit better, see [Sentry's development inclusive language docs](https://develop.sentry.dev/inclusion/#phased-out-terms). Same suggestion for the similar terms used below in this unit test.

Reply:
agreed i'll create follow up PR ok?
cause we need to change also in code https://github.com/getsentry/sentry/blob/9b5b42a0a6344be38b1e0c8742ed3411450993fb/src/sentry/relay/config/__init__.py#L121